### PR TITLE
Major changes

### PR DESCRIPTION
**Pull request by shubanms**
Added folders and sub-folders
Fixes issue #20 

# Author 
@shubanms

### CODE REVIEW

File: ALGORITHMS/Dynamic Programming/Binomial Coefficient/BinomialCoefficient.c [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
-    printf("Enter the value of n\n");
-    scanf("%d" , &n);
-    printf("Enter the value of k\n");
-    scanf("%d" , &k);
-
-	printf("Value of nCk is %d\n", binomialCoeff(n, k));
-
-	return 0;
-}
+#include <conio.h>
```
Comment (justforfunweb) [inline ALGORITHMS/Dynamic Programming/Binomial Coefficient/BinomialCoefficient.c:1]: Please add comments to explain your code

File: ALGORITHMS/Dynamic Programming/Binomial Coefficient/BinomialCoefficient.cpp [hunk 2]
```diff
@@ -1,33 +1,33 @@
...
-    
-    cout << "Enter the value of n: " << endl;
-    cin >> n;
-    cout << "Enter the value of k: " << endl;
-    cin >> k;
-
-	cout << "Value of nCk is " << binomialCoeff(n , k) << endl;
-}
-
+#include <bits/stdc++.h>
```
Comment (justforfunweb) [inline ALGORITHMS/Dynamic Programming/Binomial Coefficient/BinomialCoefficient.cpp:1]: Please add comments to explain your code

File: ALGORITHMS/Dynamic Programming/Binomial Coefficient/BinomialCoefficient.py [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
-
-        return C[n][k]
-
-if __name__ == "__main__":
-    n = int(input("Enter the value of n: "))
-    k = int(input("Enter the value of k: "))
-    print("Value of nCk is " , binom().binomialCoef(n , k))
-
-    
+class binom:
```
Comment (justforfunweb) [inline ALGORITHMS/Dynamic Programming/Binomial Coefficient/BinomialCoefficient.py:1]: Please add comments to explain your code

File: ALGORITHMS/Dynamic Programming/Fibonacci numbers/FibonacciNumbers.py [hunk 4]
```diff
@@ -1,9 +1,9 @@
-class fibo:
-    def fibonacci(self , n):
-        if n <= 1:
-            return n
-        return fibo().fibonacci(n-1) + fibo().fibonacci(n-2)
-
-
-if __name__ == "__main__":
-	print(fibo().fibonacci(int(input("Enter the number: "))))
+class fibo:
```
Comment (justforfunweb) [inline ALGORITHMS/Dynamic Programming/Fibonacci numbers/FibonacciNumbers.py:1]: Please add comments to explain your code

**Review by justforfunweb — APPROVED** [review]
- Please add comments to all your codes and explain them briefly 
- Fill in the contributions file to guide people on contributing and the rules 
- Use better variable naming convention

File: ALGORITHMS/Dynamic Programming/Binomial Coefficient/BinomialCoefficient.c (hunk 1 above)
Comment (shubanms) [reply ALGORITHMS/Dynamic Programming/Binomial Coefficient/BinomialCoefficient.c:1]: Will add them to all the files and I will create a text document in each sub-folder explaining the code in brief

File: ALGORITHMS/Dynamic Programming/Binomial Coefficient/BinomialCoefficient.py (hunk 3 above)
Comment (shubanms) [reply ALGORITHMS/Dynamic Programming/Binomial Coefficient/BinomialCoefficient.py:1]: Will add them to all the files and I will create a text document in each sub-folder explaining the code in brief

File: ALGORITHMS/Dynamic Programming/Fibonacci numbers/FibonacciNumbers.py (hunk 4 above)
Comment (shubanms) [reply ALGORITHMS/Dynamic Programming/Fibonacci numbers/FibonacciNumbers.py:1]: Will add them to all the files and I will create a text document in each sub-folder explaining the code in brief

File: ALGORITHMS/Dynamic Programming/Binomial Coefficient/BinomialCoefficient.cpp (hunk 2 above)
Comment (shubanms) [reply ALGORITHMS/Dynamic Programming/Binomial Coefficient/BinomialCoefficient.cpp:1]: Will add them to all the files and I will create a text document in each sub-folder explaining the code in brief

**Pull request merged**